languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/utils/KeywordsUtil.java

```diff
@@ -0,0 +1,36 @@
+package io.vrap.codegen.languages.javalang.client.builder.utils;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class KeywordsUtil {
```

Comment:
Hmm, ok... I agree that the java keywords don't change very often. But I would have used https://docs.oracle.com/javase/7/docs/api/javax/lang/model/SourceVersion.html, but never mind...

Reply:
I would agrEE WITH @katmatt here, java its better to delegate this task to the jdk.